subsys/net/lib/http/Kconfig

```diff
@@ -4,7 +4,7 @@
 #
 
 config HTTP
-	bool "HTTP support"
+	bool "HTTP support [Deprecated]"
```

Comment:
hmmm, we are deprecating this with what? are we deprecating the feature itself or the API? This is confusing.

Reply:
Yes, indeed confusing. Should change description to "net-app based HTTP client/server support [Deprecated]".

Reply:
This is just semantics, as there is no replacement API in the system, the HTTP support is deprecated in this case.